styles.css

```diff
@@ -356,8 +394,8 @@ body {
 /* horizontal card  */
 .card-img {
     position: relative;
-    width: 10rem;
-    height: 6rem;
+    width: 1005;
```

Comment:
for width unit  use -  vh / vw units . avoid px.

Reply:
ok